packages/table/src/regions.ts

```diff
@@ -467,6 +467,16 @@ export class Regions {
         return true;
     }
 
+    public static isRegionValidForTable(region: IRegion, numRows: number, numCols: number) {
+        if (region.rows != null && (region.rows[0] >= numRows || region.rows[1] >= numRows)) {
```

Comment:
Do we need to verify that the values are not `< 0` too, or can that not happen / is that excessive?